OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/WorkshopControllerTests.cs

```diff
@@ -647,5 +715,15 @@ private WorkshopDescriptionItemDto FakeWorkshopDescriptionItem()
                 Description = $"test description text sentence for id: {id.ToString()}",
             };
         }
+
+        private WorkshopStatusDto FakeWorkshopStatusDto(Guid workshopDtoId, WorkshopStatus workshopStatus)
```

Comment:
It is good practice to use verbs in method names to describe what the method performs. I understand that `Fake` can be used a verb but in this case it does not reflect the methods purpose. Let's rename this method to something like `WithWorkshop` or `GetWorkshop`.